### ISSUE TITLE
Including Google PageSpeed results

### ISSUE THREAD

**Issue by spencershaw** [andydavies/WPT-Bulk-Tester#14]
How do I modify the spreadsheet to include google PageSpeed scores?

**Comment by andydavies** [comment]
I'm pretty sure WPT doesn't query PageSpeed Insights anymore, so the score won't be included in the JSON results

**Comment by spencershaw** [comment]
according to
https://sites.google.com/a/webpagetest.org/docs/advanced-features/webpagetest-restful-apis
pagespeed is still an option.

On Mon, Jul 4, 2016 at 8:17 PM, Andy Davies <notifications@github.com>
wrote:

> I'm pretty sure WPT doesn't query PageSpeed Insights anymore, so the score
> won't be included in the JSON results
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/andydavies/WPT-Bulk-Tester/issues/14#issuecomment-230344017>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AMbZyCeP7Vegf8Xoo1FjcPArX_cLX1zYks5qSVxUgaJpZM4JD-Sn>
> .
>



-- 

*Spencer Shaw* | Chief Technical Officer

[image: Phone]:   020 7112 9321 | 07932 080811

[image: Email]:   spencer@streetlike.com

[image: website]:   www.streetlike.com

[image: Streetlike Logo]

WeHub, 2A Redbourne Avenue, Finchley, London, N3 2BS

**Comment by sergeychernyshev** [comment]
I don't think it ever queried the insights service, but used the same library to do the analysis.

**Comment by spencershaw** [comment]
Hi Andy,

Is there anyway from the Scenarios Tab to specify a mobile device and/or
user agent? If not, would the process to emulate mobile be to add a video
API parameter in the settings tab (in that case, how do i specify device or
user agent)?

Kind regards

Spencer
